holidays/countries/british_virgin_islands.py

```diff
@@ -0,0 +1,179 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from datetime import timedelta
+from gettext import gettext as tr
+
+from holidays.calendars.gregorian import (  # noqa: F401
+    JAN,
+    MAR,
+    JUN,
+    JUL,
+    AUG,
+    OCT,
+    DEC,
+    MON,
+    SAT,
+    FRI,
+)
+from holidays.groups import ChristianHolidays, InternationalHolidays, StaticHolidays
+from holidays.observed_holiday_base import (
+    ObservedHolidayBase,
+    SAT_TO_PREV_FRI,
+    SUN_TO_NEXT_MON,
+    MON_TO_NEXT_TUE,
+    SAT_SUN_TO_NEXT_MON,
+    ALL_TO_PREV_MON,
+    SAT_SUN_TO_NEXT_MON_TUE,
+)
+
+
+class BritishVirginIslands(
+    ObservedHolidayBase, InternationalHolidays, ChristianHolidays, StaticHolidays
+):
+    """British Virgin Islands holidays.
+
+    References:
+    * <https://en.wikipedia.org/wiki/Public_holidays_in_the_British_Virgin_Islands>
+    * <https://www.timeanddate.com/holidays/british-virgin-islands/>
+
+    * [2021] <https://bvi.org.uk/2021-public-holidays/>
+    * [2022] <http://www.bvi.gov.vg/media-centre/revised-2022-public-holidays>
+    * [2023] <https://bvi.gov.vg/media-centre/updatedofficialholidays20231>
+    * [2024] <https://bvi.gov.vg/media-centre/cabinet-approves-2024-public-holidays>
+    * [2025] <https://bvi.gov.vg/media-centre/2025-holiday-calendar-flyer-01>
+    """
+
+    country = "VG"
+    default_language = "en_VG"
+    # %s (observed).
+    observed_label = tr("%s (observed)")
+    subdivisions = ()
+    supported_languages = (
+        "en_US",
+        "en_VG",
+    )
+    start_year = 1967
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self)
+        StaticHolidays.__init__(self, BritishVirginIslandsStaticHolidays)
+        kwargs.setdefault("observed_rule", SAT_TO_PREV_FRI + SUN_TO_NEXT_MON)
+        kwargs.setdefault("observed_since", 2000)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        dt = self._add_new_years_day(tr("New Year's Day"))
+        if self._year not in {2004, 2010}:
+            self._add_observed(dt, rule=SAT_SUN_TO_NEXT_MON)
+
+        # Lavity Stoutt's Birthday.
+        self._add_observed(
+            self._add_holiday_mar_7(tr("Lavity Stoutt's Birthday")), rule=ALL_TO_PREV_MON
+        )
```

Comment:
```suggestion
        self._add_holiday_1st_mon_before_mar_7(tr("Lavity Stoutt's Birthday"))
```
Official name is "The Anniversary of the Birth of Hamilton Lavity Stoutt". I think, "Lavity Stoutt's Birthday" we can use for en_US locale.